src/Inc/Menus.php

```diff
@@ -0,0 +1,61 @@
+<?php namespace Leean\Endpoints\Inc;
```

Comment:
You're right. I forgot that.

> El 21/03/2016, a las 9:24 a.m., Adam Fenton <notifications@github.com> escribió:
> 
> In src/Inc/Menus.php:
> 
> > @@ -0,0 +1,61 @@
> > +<?php namespace Leean\Endpoints\Inc;
> Yes, makes sense. Updated to StaticApi (Static is a reserved keyword in php).
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>